src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div class="checkout-receipt">
+		<h2 class="checkout-receipt__headline">
+			Order Confirmation
+		</h2>
+		<section class="section">
+			TODO: Date<br>
```

Comment:
It looks like we'll need to add this (transaction create time) to graphql so that we can display it here. Is there a ticket for that yet?

Reply:
No ticket that I could find.

Reply:
Added CASH-1754 for this